src/pd/async/client.rs

```diff
@@ -39,11 +42,12 @@ impl RpcClient {
             })
             .collect();
 
-        let (client, members) = try!(validate_endpoints(&endpoints));
+        let env = Arc::new(Environment::new(1));
```

Comment:
if we use different Env for PD and server, we should pass different name for the Env loop thread. 

So it is better to pass name when creating Environment. @BusyJay